alibi_detect/models/tensorflow/__init__.py

```diff
@@ -11,7 +47,14 @@
     "VAE",
     "VAEGMM",
     "resnet",
+    "scale_by_instance",
```

Comment:
Just to double check; is `scale_by_instance` promoted from `tensorflow.resnet` to `tensorflow` so that we can do the `import_optional` check in `__init__`? This makes sense I think, although a minor point is that we would ordinarily want to raise a `DeprecationWarning` if the old import usage `from alibi_detect.models.tensorflow.resnet import scale_by_instance` is used. Perhaps not important in this case as `scale_by_instance` is not used a lot...

Reply:
Yeah, that's exactly correct. I'll add the relevent `DepreciationWarnings`

Reply:
see [this](https://github.com/SeldonIO/alibi-detect/pull/538#discussion_r914862049) related comment!